### ISSUE TITLE
Component for CosmosDB change feed

### ISSUE THREAD

**Issue by perokvist** [dapr/components-contrib#847]
The case of "[tailing](https://microservices.io/patterns/data/transaction-log-tailing.html)" the database, subscribing to it's "[change feed](https://docs.microsoft.com/en-us/azure/cosmos-db/change-feed)" has a lot of possibilities.
Combining a state store and tailing would open up many scenarios.
Since the building blocks for supporting this scenario might be found in *bindings* or *pubsub*, it's a matter of implementation.
There is a lot of conceptual similarities with consuming Event Hub partitions.

I don't know what kind of support different go libraries and SDK has working with the CosmosDB change feed.

But I think this kind of components have a lot of potential if possible.

**Comment by matt-lethargic** [comment]
This is something that would be great to have and probably easily created as an input binding. Having this would enable the out box pattern, from there you can do a lot of interesting things

**Comment by perokvist** [comment]
Yes, my main scenario is to be able to "subscribe" to a custom event store like;
https://github.com/perokvist/Dapr.EventStore

**Comment by yaron2** [comment]
@perokvist this can be done as an input binding for CosmosDB, which recently saw a new Go-SDK. Some work needs to be done to investigate if the SDK support change feed.

**Comment by perokvist** [comment]
@yaron2  Great news! Hope its feasible with the new SDK.
Since the change feed probably not going to support PostgreSql, maybe an input binding for LISTEN/NOTIFY could be another suggested "issue" ?
(there is only an output binding listed - https://docs.dapr.io/reference/components-reference/supported-bindings/)
